xtensa-lx-rt/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name         = "xtensa-lx-rt"
-version      = "0.16.0"
+version      = "0.16.1"
```

Comment:
Given the update breaks esp-wifi shouldn't this be `0.17.0`?

Reply:
Yeah I agree

Reply:
Yeah wasn't aware of the break when I changed that, thanks